src/app/CommandInputField.tsx

```diff
@@ -14,6 +12,9 @@ import { action } from "mobx";
 export const CommandInputField = observer((props: { controller: BottomPanelController }) => {
   const { app, logger } = getRootStore();
   const controller = props.controller;
+  let lastCmdIndex = 0;
+  let lastCmd: string[] = [];
+  let inputFocused = false;
```

Comment:
What's the reason for this variable?

Reply:
Because the hotkey of arrow up/down to get last executed command is only trigger when the input field was focused, therefore, such an boolean I believe should be added for this purpose.

Reply:
I don't think you need it, because events such as keydown events work only when the user is focusing on the input field.

Reply:
I see, Thank you. This boolean removed.